content-src/asrouter/asrouter-content.jsx

```diff
@@ -93,6 +95,15 @@ export class ASRouterUISurface extends React.PureComponent {
     this.sendImpression = this.sendImpression.bind(this);
     this.sendUserActionTelemetry = this.sendUserActionTelemetry.bind(this);
     this.state = {message: {}, bundle: {}};
+
+    if (props.document) {
+      // Create the portal element.
+      const footerContainer = props.document.getElementById("footer-snippets-container");
+      this.footerPortal = document.createElement("div");
```

Comment:
can you put this inside the `if` since you're not going to use it if `footerContainer` doesn't exist? also might be worth logging a warning here...